### ISSUE TITLE
Settings > About: app version footer is overlapping the bottom row with larger font sizes

### ISSUE THREAD

**Issue by jaclync** [woocommerce/woocommerce-ios#1122]
**Describe the bug**
When the device is on the larger font sizes, the app version footer text in `Settings > About` could overlap with other rows on certain screen sizes.

**To Reproduce**
- Go to Settings
- Tap "WooCommerce" row under `ABOUT THE APP` section
- Scroll to the bottom
- Try a few font sizes from accessibility larger font settings

**Screenshots**
![Simulator Screen Shot - iPhone 5s - 2019-07-24 at 08 13 08](https://user-images.githubusercontent.com/1945542/61755720-69797c80-adeb-11e9-9f82-f4ff8ba91ec3.png)

**Expected behavior**
Expected: the app version footer text should be shown with proper margin and not overlap with any other UI
Actual: the app version footer text overlaps with the bottom row as in screenshot

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 5s
- iOS version: iOS 12
- WooCommerce iOS version: 2.3
